src/routes/layouts/login/index.js

```diff
@@ -13,6 +13,7 @@ import Icon from 'react-fa'
 import './style.css'
 import {trackAccountModalClick, trackUserMenuOpen, trackUserMenuClose} from 'server/segment/index.js'
 import * as spaceActions from 'gModules/spaces/actions.js'
+import * as userEngine from 'gEngine/user.js'
```

Comment:
I'd prefer to do ``engine.user``, not to reach into the lib.  You may also be able to do ``import {user} from gEngine``